### PR TITLE
Set column processed after infer new predicate on it

### PR DESCRIPTION
**Pull request by terry-chelsea**
Signed-off-by: terry <fengyuatad@126.com>

## Here are some reminders before you submit the pull request
- [x] Add tests for the change
- [x] Document changes
- [x] Communicate in the mailing list if needed
- [x] Pass `make installcheck`
- [x] Review a PR in return to support the community

### CODE REVIEW

**Review by chrishajas — APPROVED** [review]
LGTM after adding a comment in the minidump. Thanks for the fix!

File: src/backend/gporca/data/dxl/minidump/InferPredicatesForProcessedColumn.mdp [hunk 1]
```diff
@@ -0,0 +1,542 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<dxl:DXLMessage xmlns:dxl="http://greenplum.com/dxl/2010/12/">
```
[the remark this answers is not in the document]
Comment (terry-chelsea) [reply src/backend/gporca/data/dxl/minidump/InferPredicatesForProcessedColumn.mdp:2]: updated

**Comment by chrishajas** [comment]
Hi @terry-chelsea, just wanted to check whether this was intentionally marked as draft or if it's good to be merged.

**Comment by terry-chelsea** [comment]
> Hi @terry-chelsea, just wanted to check whether this was intentionally marked as draft or if it's good to be merged.

Hi, i think it is good to be merged, we can set it was process just after append ned filter about one column.

**Pull request merged**